modules/vantagecloud-lake/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,149 @@
+----
+docker pull teradata/jupyterlab-extensions
+----
+Next, start a container and bind it to the existing lake-demos directory. Choose the appropriate command based on your operating system: 
+[tabs]
+====
+Windows::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $(PWD):/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+macOS::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $PWD:/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+Linux::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $PWD:/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+====
+Take note of the resulting URL and token; you’ll need them to establish the connection from Visual Studio Code.
+
+image::{dir}/terminal.png[terminal.png,align="center", width=80%]
+
+== Visual Studio Code Configuration
+Open `lake-demos` project directory in Visual Studio Code. The repository contains the following project tree: 
+
+LAKE_DEMOS
+
+* https://github.com/Teradata/lake-demos/tree/main/UseCases[UseCases]
+** https://github.com/Teradata/lake-demos/blob/main/0_Demo_Environment_Setup.ipynb[0_Demo_Environment_Setup.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/1_Load_Base_Demo_Data.ipynb[1_Load_Base_Demo_Data.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Engineering_Exploration.ipynb[Data_Engineering_Exploration.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Science_OAF.ipynb[Data_Science_OAF.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Demo_Admin.ipynb[Demo_Admin.ipynb]
+* https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]
+
+=== Edit vars.json file 
+Edit the *https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]* file to include the required credentials to run the demos +
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+| *"host"* 
+| Public IP value from your VantageCloud Lake environment
+| *"UES_URI"* 
+| Open Analytics from your VantageCloud Lake environment
+| *"dbc"*
+| The master password of your VantageCloud Lake environment.
+|====
+* You'll see that in the sample vars.json, the passwords of all users are defaulted to "password", this is just for matters of the sample file, you should change all of these password fields to strong passwords, secure them as necessary and follow other password management best practices.
+
+**To retrieve a Public IP address and Open Analytics Endpoint follow these https://quickstarts.teradata.com/vantagecloud-lake-demo-jupyter-docker.html#_create_vantagecloud_lake_environment[instructions].
```

Comment:
I don't see `**` anywhere. Why don't we move this comment directly to the table above?